src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,73 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
+    public static final String SUPPLY = "supply";
+    public static final String COMMA = ",";
+    public static final String BUY = "buy";
+    public static final String RESULT = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File fileRead = new File(fromFileName);
+        File fileWrite = new File(toFileName);
+        StringBuilder data = new StringBuilder();
```

Comment:
better move this StringBuilder to the method writeToStringBuilder and change method signature to:
`private StringBuilder writeToStringBuilder(File fileRead)`